packages/sil-voting-app/src/routes/dashboard/components/Proposal/Status.tsx

```diff
@@ -0,0 +1,106 @@
+import Block from 'medulas-react-components/lib/components/Block';
+import Img from 'medulas-react-components/lib/components/Image';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React from 'react';
+import { VoteResult } from '.';
+import deleteIcon from '../../../../assets/delete.svg';
+
+interface Props {
+  readonly expiryDate: Date;
+  readonly quorum: number;
+  readonly result: VoteResult;
+  readonly status: 'Active' | 'Submitted' | 'Ended';
+}
+
+interface VoteWinner {
+  readonly vote: string;
+  readonly percentage: number;
+}
+
+const getTotalVotes = (result: VoteResult): number => {
+  return result.yes + result.no + result.abstain;
+};
+
+const getWinner = (result: VoteResult): VoteWinner => {
+  const maxVotes = Math.max(result.yes, result.no);
+  let vote = 'Abstain';
+
+  if (maxVotes === result.yes) vote = 'Yes';
+  if (maxVotes === result.no) vote = 'No';
+  if (maxVotes === 0) vote = 'Abstain';
+
+  const percentage = Math.floor((maxVotes / getTotalVotes(result)) * 100);
+
+  return { vote, percentage };
+};
+
+const Status = (props: Props): JSX.Element => {
+  return (
+    <React.Fragment>
```

Comment:
Split this component into `<StatusEnded />` and  `<StatusPending />` then, in the Proposal/index.tsx introduce the logic for rendering one or the other.

Right now is complex to follow.